multisig/src/lib.rs

```diff
@@ -29,15 +31,15 @@ pub trait Multisig:
     /// Owner claims accumulated fees and distributes them to relayers
     /// Only centralized until we have a finalized fee distribution algorithm in place
     /// Not using get_owner API function, as the owner may also be a multisig SC
-    #[endpoint(claimAccumulatedFees)]
-    fn claim_accumulated_fees(&self, dest_address: Address) -> SCResult<()> {
+    #[endpoint(claimFees)]
+    fn claim_fees(&self, dest_address: Address) -> SCResult<()> {
         self.require_caller_owner()?;
-        require!(
-            !self.blockchain().is_smart_contract(&dest_address),
-            "Cannot have SC as destination"
-        );
 
-        self.ethereum_fee_prepay_proxy(self.ethereum_fee_prepay_address().get())
+        self.esdt_safe_proxy(self.esdt_safe_address().get())
```

Comment:
it should be distribute fee instead of this. or the esdt should be transferred to the multi sig smart contract. where the distribution can take place.
You could use the deposit function = maybe from multisig